config/checkers.yaml

```diff
@@ -40,3 +40,6 @@ parameters:
 
     # Spryker dev packages checker
     spryker_dev_packages_checker_doc_url: https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/spryker-dev-packages-checker.html
+
+    # Discouraged packages checker
+    discouraged_packages_checker_doc_url: https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/discouraged_packages_checker.html
```

Comment:
oh crap. This `Discouraged` naming sneaked into the code. This naming is so confusing. Too bad we couldn't make up something better.